docs/tutorials/index.rst

```diff
@@ -0,0 +1,49 @@
+GeoMesa |release| Tutorials
+===========================
+
```

Comment:
I should also mention that the existing tutorials on the website will almost certainly be going away and replaced with these when we push up the release documentation. This will allow us to make sure that the tutorials stay up-to-date (by being parallel with the main documentation), and allow them to directly link to sections of the manual rather than saying things like "make sure you have done the deployment tutorial first". These are already newer than the online tutorial in that they point to geomesa-tutorials, as mentioned above.

That having been said we could make routes on the site that make sure that paths like http://geomesa.org/tutorials/geomesa-kafka-quickstart still resolve, but would just point to the version of the tutorial in the documentation tree.